backend/app/services/subscription/follow_service.py

```
@@ -61,6 +63,79 @@
+            raw_display_name.strip()
+            if isinstance(raw_display_name, str) and raw_display_name.strip()
+            else subscription.name
+        )
+
+        raw_description = spec.get("description")
+        description = raw_description if isinstance(raw_description, str) else None
+
+        raw_visibility = spec.get("visibility", SubscriptionVisibility.PRIVATE.value)
+        try:
+            visibility = SubscriptionVisibility(raw_visibility)
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)